src/jukebox/jukebox/rpc/server.py

```diff
@@ -107,7 +107,7 @@ def run(self):
         while self._keep_running:
             # Wait for next request from client
             message = self.socket.recv()
-            nt = nanotime.now().nanoseconds()
+            nt = time.time_ns()
```

Comment:
This was originally intended to measure the time delay in the rpc calls back in the first trail code.
Not sure if we still want to keep this.

Anyhow, it seams that time.time_ns() is not the same as nanotime.
My understanding was that nanotime is a different system call, which can be used across different languages. I think it would require some investigation to prove the correct function when switching to time_ns()

Reply:
I think, we could safely switch to time_ns() (or remove the time measuering altogether). If we pull any of these two options, we should remove nanotime from the requirements.txt as well :-)

Reply:
I would vote for removing it as the prove of concept and prove of faster reaction times had been given some time ago already. Otherwise it will lead to continuous confusion about the purpose.

Reply:
I vote for removing as well ;)